### DISCUSSION TITLE
Who is using Spring Boot Admin?

### DISCUSSION THREAD

**Discussion by joshiste** [codecentric/spring-boot-admin#3756] [General]
I'm curious about who is using the Spring Boot Admin. Would be nice if you make a short comment in this issue with your company name (and a approximate numbers of services you monitor)

**Comment by dl1ely** [comment]
Using it in a Kubernetes based deployment of roughly 20 Spring Boot Microservices. Thank you very much! http://zontal.io

**Comment by mknutty** [comment]
My company is using it and I have used it elsewhere.  We have it in 4 environments with 18 apps  (and growing as we convert / create new).

**Comment by rittermit** [comment]
I'm using it to keep track of my 53 small(isch) apps - mostly deployed on tomcat in 3 environments - more to come.

**Comment by skosijer** [comment]
We are using it at [Levi9](https://www.levi9.com/). On a large distributed system with over 40 running MS. Keep up the good work! 

From Novi Sad with <3

**Comment by ingogriebsch** [comment]
We are using it as part of our [application](https://www.gbtec.de/en/bpm-software-en/bic-cloud-bpm-suite/) wide monitoring in docker based deployments of roughly 30 Spring Boot services. Thanks for all the effort. Keep up the good work! :)

**Comment by MartinDevillers** [comment]
We have multiple teams at [DPG Media](https://www.dpgmediagroup.com/?q=en) use it to manage about 50 micro-services and front-ends running in [AWS ECS Fargate](https://aws.amazon.com/fargate/).

**Comment by bjoernb94** [comment]
I work in the public sector. And we use the tool to monitor our application software with it.

**Comment by rainerfrey** [comment]
Inxmail GmbH is using separate instances of Spring Boot Admin in 3 environments (dev, staging, production), each with ~25 applications (up to ~45 instances) deployed in Kubernetes.

**Comment by HRPatel2000** [comment]
@Aventiv Technologies, we use Spring Boot Admin to monitor ~20 Microservices in 7 different environments.

**Comment by cash1981** [comment]
We Gjensidige Bank, a part of Nordea is using it. We have around 20 services we are monitoring. We are running it in cluster with 2 instances replicated in dev/test and prod environment

**Comment by claitonp** [comment]
Brazilian Post Company is using separate instances of Spring Boot Admin in 3 environments (development, approval, production), each with ~ 50 applications deployed on Kubernetes. We expect to reach 200 applications next year in each environment.
Congratulations for the work!

**Comment by retevastum** [comment]
Pretty much the largest logistics company available.

**Comment by sabareeshkkanan** [comment]
27 ms in 4 different environment. Finance firm

**Comment by mehiel** [comment]
Almost all of Logicea's clients. From small startups to National Telcos to Gov.

**Comment by enesify** [comment]
We use Spring Boot Admin to monitoring approximately 10 Spring Boot services (increases day by day :) ) in 2 diffrent environments at Turkish Airlines Technic Inc. Thank you for this great tool.

**Comment by arunkpatra** [comment]
I use SBA in a complex IoT platform to monitor multiple microservices.

**Comment by fabiosakiyama** [comment]
Just started using SBAdmin with kubernetes, at the end of the implementation we will probably have ~ 20 apis registered :)

**Comment by sepgh** [comment]
We are using it @youtopin (in our closed source projects for production) with eureka auto discovery to monitor our services and get discord notifications.

**Comment by arunkpatra** [comment]
@joshiste Just launched new project and I use SBA - https://github.com/arunkpatra/thingverse

**Comment by mle-enso** [comment]
It's [idealo](https://www.idealo.de) and even only in my team we use it for 10+ services, much more in the whole company.

**Comment by piercer** [comment]
We are an investment bank and we have multiple teams using SBA to monitor many different services. It is indispensable - thank you

**Comment by guerricmerleHUG** [comment]
We (Hôpital Universitaires de Genève) used SBA in docker swarm to monitor ~30 SpringBoot services

**Comment by zhhaojie** [comment]
www.iboxpay.com widely use SBA

**Comment by Docjones** [comment]
German logisitc network running 15+ services
![grafik](https://user-images.githubusercontent.com/561429/92752131-f0d7d580-f388-11ea-9f4c-770aae20f636.png)

**Comment by taasjord** [comment]
We use it in a European airline, toghether with Eureka server. We currently have 58 applications registered in three enviroments.

**Comment by aumer-amr** [comment]
Currently using it for a charity backend system, at the moment 6 services, according the roadmap should be 15+

**Comment by Toparvion** [comment]
We're actively using Spring Boot Admin for monitoring from 45 to 100+ applications in 3 environments at [CFT](https://github.com/cft-group).

**Comment by Xyaren** [comment]
Using it for an internal stack at TUI a germany based international travel agency.

**Comment by liran2000** [comment]
Using it at [Mend](https://www.mend.io/) (formerly WhiteSource) for a few dozens of multi-instances applications.

**Comment by maks-pain** [comment]
Kyivstar, DMP department
 Prod:
![image](https://user-images.githubusercontent.com/11442372/192555950-26edb323-c107-4ea7-a2df-57c6db34576d.png)

**Comment by MichaelZett** [comment]
Monitoring 85+ Services at [EOS](https://de.eos-solutions.com/) in Hamburg, Germany.

**Comment by pgehl** [comment]
It's our main observability entry point (we added custom links to logs, grafana, k8s dashboard etc) at Ludia a mobile video game studio.
We use sba to monitor 30+ micro-services in 3 k8s clusters, with several namespaces each representing around 100-150 instances.

**Comment by bitbrain** [comment]
[Global Media Group](https://global.com/) (London based)

**Comment by ahoehma** [comment]
Siemens is also using sba :-)

![image](https://github.com/codecentric/spring-boot-admin/assets/548214/eeb24e00-5c7e-4ed3-866a-cdce68839c30)

**Comment by TrueBurn** [comment]
Work in finance. I plan to use it for +250 services with over 3000 pods. I built something bespoke to call the refresh endpoint on all instances for a service in k8s but looking for something I don't have to maintain myself. I was also looking for a Spring Boot version of [HealthCheckUI](https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks) that C# has when I found this. Actually looking for a way now to change logger level for all instances of a service instead of per pod.

Note: I will most likely deploy an instance per namespace in each of the clusters for performance and security reasons,

**Comment by cdprete** [comment]
In SIX we're starting to build our monitor around it.

**Comment by kdejaeger** [comment]
I decided to use sba at [Surf ShareControl](https://www.surf.nl/en/services/publishing/sharecontrol). We have around 70 institutions to harvest from and this tool gives me a nice overview of the status of each.
